sklearn/neighbors/base.py

```diff
@@ -563,14 +647,17 @@ def radius_neighbors_graph(self, X, radius=None, mode='connectivity'):
         --------
         kneighbors_graph
         """
-        X = check_array(X, accept_sparse=['csr', 'csc', 'coo'])
+        # radius_neighbors does the None handling.
+        if X is not None:
+            X = check_array(X, accept_sparse='csr')
```

Comment:
Do we have any idea why CSC/COO were previously supported? I think since calculations are row-wise, constraining to CSR makes sense, but changing it should not be arbitray.